introduction_to_applying_machine_learning/README.md

```diff
@@ -14,3 +14,4 @@ These examples provide a gentle introduction to machine learning concepts as the
 - [Fair Linear Learner](fair_linear_learner) is an example of an effective way to create fair linear models with respect to sensitive features.
 - [Population Segmentation of US Census Data using PCA and Kmeans](US-census_population_segmentation_PCA_Kmeans) analyzes US census data and reduces dimensionality using PCA then clusters US counties using KMeans to identify segments of similar counties.
 - [Document Embedding using Object2Vec](object2vec_document_embedding) is an example to embed a large collection of documents in a common low-dimensional space, so that the semantic distances between these documents are preserved.
+- [Traffic violations forecasting using DeepAR](deepar_chicago_traffic_violations) is an example to use daily traffic violation data to predict pattern and seasonality using Amazon DeepAR alogorithm.
```

Comment:
Minor style comment: "to use" -> "of using"

Reply:
fixed